lib/interfaces/uncaught.js

```diff
@@ -47,14 +46,7 @@ function uncaughtExceptionHandler(client, config, err) {
 
   errorHandlerRouter(err, em);
 
-  if (config.onUncaughtException === uncaughtHandlingOptions.report) {
-
-    client.sendError(em);
-  } else if (config.onUncaughtException ===
-             uncaughtHandlingOptions.reportAndExit) {
-
-    client.sendError(em, handleProcessExit);
-  }
+  client.sendError(em, handleProcessExit);
```

Comment:
You no longer need `config` in this function, so you can remove it from the signature.